src/main/resources/io/snyk/jenkins/SnykReportBuildAction/index.jelly

```diff
@@ -8,12 +8,8 @@
 
     <l:main-panel>
       <h3>Reports</h3>
-      <j:forEach var="artifact" items="${it.run.artifacts}">
-        <j:if test="${artifact.toString().indexOf('snyk_report.html') != -1}">
-          <a href="report#${artifact}">${artifact.toString().replace("_snyk_report.html", "")}</a>
-          <br/>
-        </j:if>
-      </j:forEach>
+      <a href="./report">View report</a>
```

Comment:
This page just links to one report. Previously we went through all the artifacts, but we only ever had one artifact. We could remove this page and just show the single report but we'll be supporting multiple reports soon so I kept it as it is.